### PR TITLE
Add particle source

### PR DESCRIPTION
**Pull request by itziakos**
This PR adds a mayavi source for a CUDS Particles container.

The current implementation supports:
- Particles
- Bonds

A basic example is also provided that shows the particle in mayavi2.

### CODE REVIEW

**Comment by itziakos** [comment]
@dpinte, do you have time for a review

File: simphony_mayavi/sources/particles_source.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from mayavi.sources.vtk_data_source import VTKDataSource
+from tvtk.api import tvtk
+from traits.api import Dict
+
+
+class ParticleSource(VTKDataSource):
+    """ A Mayavi Source wrapping a SImphony CUDS Particle container.
```
Comment (pberkes) [inline simphony_mayavi/sources/particles_source.py]: `SImphony` with lowercase `I`?

File: simphony_mayavi/sources/particles_source.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            lines.append([point2index[uid] for uid in bond.particles])
+
+        data = tvtk.PolyData(points=points, lines=lines)
+        return cls(
+            data=data,
+            point2index=point2index,
+            bond2index=bond2index)
+
+
+def cell_array_slicer(data):
```
Comment (pberkes) [inline simphony_mayavi/sources/particles_source.py:37]: docstring missing

**Comment by pberkes** [comment]
Apart from two comments regarding docstrings, LGTM

File: simphony_mayavi/sources/particles_source.py (hunk 1 above)
Comment (itziakos) [inline simphony_mayavi/sources/particles_source.py]: good catch, It should be actually `SimPhoNy`

**Comment by itziakos** [comment]
Thanks @pberkes

**Pull request merged**